flyte.config

```diff
@@ -0,0 +1,11 @@
+[sdk]
+workflow_packages=workflows.getting_started
+python_venv=flytekit_venv
+
+[auth]
+assumable_iam_role=arn:aws:iam::12345:role/flyteworkflows-iad
+raw_output_data_prefix=s3://my-s3-bucket/cookbook
+
+[platform]
+url=localhost:30081
```

Comment:
wrong, if this is not used, lets delete it please

Reply:
it's a required arg. @kumare3 do we want to make it optional in the flyte config? I think we should

Reply:
I agree. otherwise the value is wrong

Reply:
updated flytekit